quiz.js

```diff
@@ -0,0 +1,46 @@
+// Create a tree function that accepts a single object as an argument.
+// The object has two key/value pairs from user input in a <input type="text"> field in the DOM.
+// Tree should be shown in the console.
+
+function tree() {
+  var height = document.getElementById("height").value;
+  var char = document.getElementById("char").value;
+
+  for (var i = 0; i < height; i++) {
```

Comment:
you will need to change these variables to match the structure of the tree object